### PR TITLE
Add memory randomization runs

### PR DESCRIPTION
**Pull request by DrewScoggins**
This is adding it to run against all tests, but only in one single configuration. We are doing this to collect data that we can then use to test the visualization system.

### CODE REVIEW

File: eng/testing/performance/performance-setup.ps1 [hunk 1]
```diff
@@ -136,6 +139,10 @@ if ($NoR2R) {
     $SetupArguments = "$SetupArguments --no-r2r"
 }
 
+if ($MemoryRandomization) {
+    $SetupArguments = "$SetupArguments --memory-randomization"
+}
+
```
Comment (cincuranet) [inline eng/testing/performance/performance-setup.ps1:145]: This is not needed.

File: eng/testing/performance/performance-setup.sh [hunk 2]
```diff
@@ -385,8 +385,16 @@ if [[ "$nor2r" == "true" ]]; then
     configurations="$configurations R2RType=nor2r"
 fi
 
+if [[ "$memoryrandomization" == "true" ]]; then
+    configurations="$configurations MemoeryRandomization=true"
+    extra_benchmark_dotnet_arguments="$extra_benchmark_dotnet_arguments --memory-randomization true"
+fi
+
```
Comment (cincuranet) [inline eng/testing/performance/performance-setup.sh:392]: This is not needed.

**Review by cincuranet — CHANGES_REQUESTED** [review]
I guess the changes in YAML files will be fixed, it was only for testing, right?

**Review by LoopedBard3 — APPROVED** [review]
Looks good, can't wait to see the results of the experiment. 👍

**Review by cincuranet — APPROVED** [review]

**Pull request merged**